block/manager.go

```diff
@@ -372,30 +376,30 @@ func NewManager(
 		headerBroadcaster: headerBroadcaster,
 		dataBroadcaster:   dataBroadcaster,
 		// channels are buffered to avoid blocking on input/output operations, buffer sizes are arbitrary
-		headerInCh:                  make(chan NewHeaderEvent, eventInChLength),
-		dataInCh:                    make(chan NewDataEvent, eventInChLength),
-		headerStoreCh:               make(chan struct{}, 1),
-		dataStoreCh:                 make(chan struct{}, 1),
-		headerStore:                 headerStore,
-		dataStore:                   dataStore,
-		lastStateMtx:                new(sync.RWMutex),
-		lastBatchData:               lastBatchData,
-		headerCache:                 cache.NewCache[types.SignedHeader](),
-		dataCache:                   cache.NewCache[types.Data](),
-		retrieveCh:                  make(chan struct{}, 1),
-		daIncluderCh:                make(chan struct{}, 1),
-		logger:                      logger,
-		txsAvailable:                false,
-		pendingHeaders:              pendingHeaders,
-		pendingData:                 pendingData,
-		metrics:                     seqMetrics,
-		sequencer:                   sequencer,
-		exec:                        exec,
-		da:                          da,
-		txNotifyCh:                  make(chan struct{}, 1), // Non-blocking channel
-		signaturePayloadProvider:    managerOpts.SignaturePayloadProvider,
-		validatorHasherProvider:     managerOpts.ValidatorHasherProvider,
-		namespaceMigrationCompleted: &atomic.Bool{},
+		heightInCh:                         make(chan NewHeightEvent, eventInChLength),
```

Comment:
The events in this channel are not persisted. On node restart, the state must be restored in some way. Not sure if there is a mechanism covering this already

Reply:
cache added.